ui/templates/header.html

```diff
@@ -10,11 +10,9 @@
             Sign out
           </a>
           {% else %}
-          <div class="members-login">
-            <a href="{% url 'social:begin' 'edxorg' %}">Log In</a>
-          </div>
+          <div class="members-login"><a class="btn ghost upper-case" href="{% url 'social:begin' 'edxorg' %}">Log In</a></div>
           <a class="btn btn-login white-button open-signup-dialog upper-case">
-            Sign up
+            Sign Up
```

Comment:
This change does not do anything, since all letters are capital. Using `upper-case` here.